vertx-web-api-contract/pom.xml

```diff
@@ -62,6 +62,7 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+
```

Comment:
that's done on purpose to trigger this action right ?

Reply:
yep will remove it